server/src/immich/controllers/audit.controller.ts

```diff
@@ -15,4 +24,22 @@ export class AuditController {
   getAuditDeletes(@AuthUser() authUser: AuthUserDto, @Query() dto: AuditDeletesDto): Promise<AuditDeletesResponseDto> {
     return this.service.getDeletes(authUser, dto);
   }
+
+  @AdminRoute()
+  @Get('file-report')
+  getAuditFiles(): Promise<FileReportDto> {
+    return this.service.getFileReport();
+  }
+
+  @AdminRoute()
+  @Post('file-report/checksum')
```

Comment:
Are we using `Post` here because we need the body?

Reply:
Correct, we send a list of files to request checksums for